lib/kitchen/directions/bake_notes.rb

```diff
@@ -46,7 +46,7 @@ def self.v1(book:)
             note.prepend(child:
               <<~HTML
                 <h3 class="os-title" data-type="title">
-                  <span class="os-title-label">#{note.autogenerated_title}</span>
+                  <span class="os-title-label">#{note.has_class?('media-2') ? 'Media' : note.autogenerated_title}</span>
```

Comment:
just FYI university physics has a note with the `media-2` class, except the title is `Interactive` 👀

Reply:
With book-specific translations the `Media` title should move into the `/books/calculus/locales/en.yml` translation for the `media-2` key.